src/NDItk/depletion/ReactionMultiplicities/src/read.hpp

```diff
@@ -0,0 +1,39 @@
+  /**
+   *  @brief Read the record data
+   *
+ *  @param[in,out] iter        an iterator to the current position in the input
+ *  @param[in,out] end         an iterator to the end of the input
+ *  @param[in,out] reactions   the number of reactions
+   */
+  template< typename Iterator >
+  void read( Iterator& iter, const Iterator& end, std::size_t reactions ) {
+
+    // read in the data
+    this->reactions_ = reactions;
+    std::vector< int > data;
+    while ( reactions-- ) {
+
+      data.push_back( njoy::tools::disco::FreeFormatInteger::read< int >( iter, end ) );
```

Comment:
Would you mind adding a comment as to what data this should be reading? Like, the first is the... MT? Second the number of products?

Reply:
Changed in fix/review-part6